gdcdatamodel/models/__init__.py

```diff
@@ -593,7 +596,7 @@ def load_edges():
     for src_cls in Node.get_subclasses():
         cache_case = (
             not src_cls._dictionary['category'] in NOT_RELATED_CASES_CATEGORIES
-            or src_cls.label in ['annotation']
+            or src_cls.get_label() in ['annotation']
```

Comment:
we can change this and other instances back to `.label` since we merged [this](https://github.com/NCI-GDC/psqlgraph/pull/174)